Public Dashboards: Change custom branding public dashboard footer behavior (#81831)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md

;header_logo_hide =
```

> `footer_hide` is no longer available. Set either a `footer_text` or `footer_logo`. Otherwise, the default Grafana footer is displayed.

Reviewer: I think you should add the `breaking-change` label to this PR and add this as the deprecation notice.

I don't think we should mention a removed option from the documentation so this should be replaced by: 
```
You can set either the `footer_text`, the `footer_logo`, or both. Otherwise, the default Grafana footer is displayed.
```

Author: How do I add deprecation notice? Is it same as breaking change notice in the pr ([ex](https://github.com/grafana/grafana-enterprise/pull/6328)?

Author: Also this is an enterprise only feature right? Should the notice/label be in the [enterprise pr](https://github.com/grafana/grafana-enterprise/pull/6235) instead?

> `footer_hide` is no longer available. Please set either a `footer_text` or a `footer_logo`; else it will default to the Grafana footer.

Author: @imatwawana , can you please look over the verbage?

Reviewer 2: ```suggestion
`footer_hide` is no longer available. Set either a `footer_text` or `footer_logo`. Otherwise, the default Grafana footer is displayed.
```
